vue-frontend/src/components/contact/ContactForm.vue

```diff
@@ -79,7 +83,12 @@
                   :disabled="disableInput"
                 />
                 <div
-                  class="hover:tw-text-white v2-normal-3-text contact-button tw-flex tw-items-center tw-border tw-border-black-900 tw-rounded-lg tw-px-6 tw-py-2 hover:tw-bg-black-900 tw-mb-6 sm:tw-mr-9 sm:tw-mb-0 active:tw-scale-[0.98]"
+                  :class="
+                    contactType == 2
+                      ? 'hover:tw-text-black-900 tw-bg-black-900 tw-text-white hover:tw-bg-white'
+                      : 'hover:tw-text-white  hover:tw-bg-black-900 '
+                  "
+                  class="v2-normal-3-text contact-button tw-flex tw-items-center tw-border tw-border-black-900 tw-rounded-lg tw-px-6 tw-py-2 tw-mb-6 sm:tw-mr-9 sm:tw-mb-0 active:tw-scale-[0.98]"
```

Comment:
Why do we need scale?

Reply:
we have given an active effect on the all button click on our website, so that's why i added scale